src/model/keychain.ts

```diff
@@ -111,20 +138,20 @@ export async function getAllKeysAnonymized() {
   return data;
 }
 
-export async function loadAllKeysOnly(authenticationPrompt) {
+export async function loadAllKeysOnly(): Promise<null | UserCredentials[]> {
```

Comment:
I think for this guy, it returns <null | string[]> instead of UserCredentials[]

Reply:
Each element of the array is an object `{ username }` which now conforms with UserCredentials (after my fix)